css/mobile.css

```diff
@@ -0,0 +1,386 @@
+/* mobile styles */
+
+.menu {
+  margin-top: 10px;
+}
+
+.nav {
+  margin-bottom: 50px;
+}
+
+.polygons {
+  position: relative;
+}
+
+.polygon,
+.triangle {
+  width: 74px;
+  height: 74px;
+}
+
+.polygon {
+  clip-path: polygon(0% 0%, 100% 0, 50% 50%, 0 100%, 0% 100%);
+}
+
+.triangle {
+  background-color: #e45865;
+  border-color: #e35461;
+  clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+  position: absolute;
+  left: 35px;
+}
+
+.rectangle {
+  width: 20px;
+  height: 2px;
+  background: #c4c4c4;
+  margin-top: 2px;
+}
+
+.header_name,
+.project {
+  margin-left: 16px;
+  font-size: 50px;
+  font-weight: 400;
+  font-family: 'Playfair Display', sans-serif;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.p_text {
+  margin: 20px 16px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.collab-btn,
+.form-btn {
+  background: #1a2236;
+  font-weight: bold;
+  font-size: 16px;
```

Comment:
`font-size: 16px` is already specified in the body element which means by default all other elements will inherit this font size, therefore specifying it again here is redundant.

Reply:
ok, noted.